src/content/docs/apis/nerdgraph/nerdgraph-usage-limits.mdx

```diff
@@ -33,5 +33,7 @@ If you're making concurrent requests to NerdGraph in your code it's important th
  * Implement a pooling solution to ensure requests are only made when a concurrent connection is detected as being available.
```

Comment:
Since pooling is the preferred solution, we could make this bullet point here on line 33 the first bullet point. You could also redundantly link `a pooling solution` to your tutorial (no harm in having multiple references to it imo!)

Reply:
Changed order of bullets